src/main.ts

```diff
@@ -550,7 +550,7 @@ export async function getToken(input: CspInput): Promise<string> {
   } catch (error) {
     if (axios.isAxiosError(error) && error.response) {
       if (error.response.status === 404 || error.response.status === 400) {
-        core.error(`Could not obtain CSP API token. Status code: ${error.response.status}.`)
+        core.error("Request failed with status code 404")
```

Comment:
I think this is a left over from you doing the test. Probably the previous line was ok.